pkg/app/build.go

```diff
@@ -106,12 +112,23 @@ func build(clicontext *cli.Context) error {
 	logger := logrus.WithField("builder-options", opt)
 	logger.Debug("starting build command")
 
-	builder, err := buildutil.GetBuilder(clicontext, opt)
-	if err != nil {
-		return err
-	}
-	if err = buildutil.InterpretEnvdDef(builder); err != nil {
-		return err
+	platforms := strings.Split(opt.Platform, ",")
+	for _, platform := range platforms {
+		o, p := opt, platform
+		o.Platform = p
+		if len(platforms) > 1 {
+			o.Tag += "-" + strings.Replace(p, "/", "-", 1)
+		}
```

Comment:
What does this do? Can you add some comments?

Reply:
build images with same tag could cause image overwriting, so here I add an platform suffix to differentiate them.

Reply:
This needs to be documented in CLI help. We'd better create a new issue to track it.